ansible/roles/ghostunnel/defaults/main.yml

```diff
@@ -0,0 +1,225 @@
+
+                                                                   # ]]]
+                                                                   # ]]]
+# SSL configuration [[[
+# ---------------------
+
+# .. envvar:: ghostunnel__pki [[[
+#
+# Enable or disable support for SSL in Ghostunnel (using :ref:`debops.pki`).
+# You can not use Ghostunnel role with pki disabled.
+ghostunnel__pki: '{{ True
+                     if (ansible_local.pki.enabled|d()|bool and
+                         ghostunnel__pki_realm in ansible_local.pki.known_realms)
+                     else False }}'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__pki_path [[[
+#
+# Base path for PKI directory.
+ghostunnel__pki_path: '{{ ansible_local.pki.base_path|d("/etc/pki") }}'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__pki_realm [[[
+#
+# Default PKI realm used by Ghostunnel servers and clients.
+ghostunnel__pki_realm: '{{ ansible_local.pki.realm|d("domain") }}'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__pki_ca [[[
+#
+# Root CA certificate used by Ghostunnel servers and clients, relative to :envvar:`ghostunnel__pki_realm`.
+ghostunnel__pki_ca: 'CA.crt'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__pki_crt [[[
+#
+# Host certificate used by Ghostunnel servers and clients, relative to :envvar:`ghostunnel__pki_realm`.
+ghostunnel__pki_crt: 'default.crt'
+
+                                                                   # ]]]
+# .. envvar:: ghostunnel__pki_key [[[
+#
+# Host private key used by Ghostunnel servers and clients, relative to :envvar:`ghostunnel__pki_realm`.
+ghostunnel__pki_key: 'default.key'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# Configuration for other Ansible roles [[[
+# -----------------------------------------
+
+# .. envvar:: ghostunnel__golang__dependent_packages [[[
+#
+# Configuration of the :ref:`debops.golang` role.
+ghostunnel__golang__dependent_packages:
+
+  - name: 'ghostunnel'
+    state: 'present'
+    upstream_type: 'url'
+    url:
+
+      - src: 'https://github.com/ghostunnel/ghostunnel/releases/download/{{ ghostunnel__release_version }}/ghostunnel-{{ ghostunnel__release_version }}-linux-amd64-with-pkcs11'
+        dest: 'releases/linux-amd64/ghostunnel/ghostunnel/{{ ghostunnel__release_version }}/ghostunnel-{{ ghostunnel__release_version }}-linux-amd64-with-pkcs11'
+        checksum: 'sha256:{{ ghostunnel__release_checksum }}'
+
+    url_binaries:
+
+      - src: 'releases/linux-amd64/ghostunnel/ghostunnel/{{ ghostunnel__release_version }}/ghostunnel-{{ ghostunnel__release_version }}-linux-amd64-with-pkcs11'
+        dest: 'ghostunnel'
+        notify: [ 'Restart ghostunnel services' ]
```

Comment:
Did you consider building the `ghostunnel` binary from cloned source code as an alternative approach? The project tags are signed with GPG keys, that could be a better base for verification.

Reply:
Ghostunnel requires Go 1.12 or later to build. golang-go buster backport it's ok, is 1.14, but stretch backport is 1.11. So, I could allow install from source but only for buster, ok?

Reply:
Sure. Debian Stretch is oldstable, you don't need to care about role compatibility with it, unless you want to deploy `ghostunnel` specifically on Stretch, although I wouldn't see a reason. When you write a new role, focus on current Debian Stable. :)

Reply:
I can't install the GPG public key from keyserver network or keybase.io, so I've opened a issue in `ghostunnel` project:
https://github.com/ghostunnel/ghostunnel/issues/326

I've uploaded a version that it installs GPG public key from data with a small fix in keyring role for allow it.

But if the author of `ghostunnel` publish her/his GPG public key, i could revert this changes.

Reply:
I have sended de GPG public key to the keyserver:

```
gpg --keyserver hkp://pool.sks-keyservers.net --send-key 2C67955A411CD552810E7598CE8B85D91C004E26
```
And also i have uploaded changes. So, now you can review this part.